### DISCUSSION TITLE
"Public Sponsor" Badge Suggestions

### DISCUSSION THREAD

**Discussion by ajmeese7** [community/community#31354] [Other Feature Feedback, Questions, & Ideas]
I have several notes and suggestions about the "Public Sponsor" profile badge:

1. Whenever you sponsor a new person, the badge currently replaces your "1st sponsorship" with them. I have sponsored various individuals over the years, but my "1st sponsorship" is now someone that I have only sponsored for a month.
2. Past sponsorships are not included, so "@ajmeese7 is sponsoring 4 organizations or users." does not acknowledge the other users I have supported over the years that I am not a current sponsor of.
    - I would like to see the message change to something along the lines of "@ajmeese7 is currently sponsoring 4 organizations or users and has sponsored a total of 11."
3. One idea is the inclusion of a tier system like with many of the other badges for total users/organizations sponsored. Tiers could be 1, 4, 16, and 64 users or something along those lines, to give additional recognition if a user has been financially supporting a large number of other users. This could also be for number of months or something, it does not necessarily have to be on a per-user/org basis.
4. Having a display of the profile pictures of all the users/orgs that a user has financially supported over the years, possibly with a hover feature indicating the total months sponsoring that person and the overall monetary contribution.

**Comment by paradox-bd** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Reply by ajmeese7** [reply]
Help with what ?

**Comment by Mirinda4511-bit** [comment]
> I have several notes and suggestions about the "Public Sponsor" profile badge:
> 
> 1. Whenever you sponsor a new person, the badge currently replaces your "1st sponsorship" with them. I have sponsored various individuals over the years, but my "1st sponsorship" is now someone that I have only sponsored for a month.
> 2. Past sponsorships are not included, so "@ajmeese7 is sponsoring 4 organizations or users." does not acknowledge the other users I have supported over the years that I am not a current sponsor of.
>     - I would like to see the message change to something along the lines of "@ajmeese7 is currently sponsoring 4 organizations or users and has sponsored a total of 11."
> 3. One idea is the inclusion of a tier system like with many of the other badges for total users/organizations sponsored. Tiers could be 1, 4, 16, and 64 users or something along those lines, to give additional recognition if a user has been financially supporting a large number of other users. This could also be for number of months or something, it does not necessarily have to be on a per-user/org basis.
> 4. Having a display of the profile pictures of all the users/orgs that a user has financially supported over the years, possibly with a hover feature indicating the total months sponsoring that person and the overall monetary contribution.
